data/presets/highway/track.json

```diff
@@ -30,29 +30,34 @@
     "terms": [
         "4wd",
         "4x4",
-        "agricultural road",
         "atv",
-        "carriage road",
         "double track",
-        "farm road",
-        "fire road",
-        "forest road",
         "four wheel drive",
         "high-clearance",
         "jeep",
-        "land access road",
-        "logging road",
         "offroad",
+        "off-road",
         "overgrown",
         "primitive",
         "quad",
-        "ranch road",
         "rut",
-        "track",
-        "track road",
-        "two track",
-        "unmaintained",
-        "woods road"
+        "unmaintained"
     ],
-    "name": "Track / Land-Access Road"
+    "name": "Track / Land-Access Road",
+    "aliases": [
+        "Agricultural Road",
+        "Carriage Road",
+        "Dirt Road",
+        "Land-Access Road",
+        "Farm Road",
+        "Fire Road",
+        "Forest Road",
+        "Forestry Road",
+        "Logging Road",
+        "Ranch Road",
+        "Track Road",
+        "Two-Track Road",
+        "Unmaintained Road",
+        "Woods Road"
+    ]
```

Comment:
“Vehicular Trail” would be a good alias, if not a good candidate for the preset’s name. I had previously overlooked this term in all the fuss about maintenance. Sometimes it’s a synonym of “4×4 road“, but it doesn’t emphasize access restrictions or surface quality.